jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -72,6 +72,7 @@
               export PROJECT="k8s-jkns-e2e-gce-federation"
               export FEDERATION=true
               export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+              export FEDERATIONS_DOMAIN_MAP="federation=ubernetes-test.com"
```

Comment:
It is not a domain that we own.  It is probably better to use a RFC 2606-compliant domain name here (and omit the "ubernetes" part, as we're trying to discourage the use of that slang term in favor of "Cluster Federation".

https://tools.ietf.org/html/rfc2606#page-2

e.g. 

$ gcloud dns managed-zones create federation-dot-test --description "Test domain for Kubernetes cluster federation" --dns-name "federation.test"
Created [.../managedZones/federation-dot-test].
NAME                 DNS_NAME          DESCRIPTION
federation-dot-test  federation.test.  Test domain for Kubernetes cluster federation